apps/web/src/components/Publication/OpenAction/UnknownModule/Decent/CreatePublicationEmbed.tsx

```diff
@@ -0,0 +1,124 @@
+import type { Nft, OG } from '@hey/types/misc';
```

Comment:
TBH we should not have a different component for preview, if you check swap oa, we have single component both for preview and render on publication